.github/workflows/scheduled.yml

```diff
@@ -25,15 +25,17 @@ jobs:
           HYPOTHESIS_PROFILE: ci
         run: tox -e hypothesis-slow
   devel_tests:
-    name: unittest / 3.13-dev
+    name: unittest / 3.14-dev
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Install Python 3.13-dev
+      - name: Install Python 3.14-dev
         uses: actions/setup-python@v5
         with:
-          python-version: 3.13-dev
+          # select latest version here:
+          # https://github.com/actions/python-versions/blob/main/versions-manifest.json
+          python-version: 3.14.0-alpha.0
```

Comment:
Don't we want to just use 3.14-dev alias per suggestion here https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input?   Or are you explicitly trying to pin the version so we don't get spurious errors as alpha migrates to alpha2 etc?